src/test/functional/features/opening-hours.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Opening Hours
+
+  Background:
+    Given I am on new browser
+    Given I am on FACT homepage
+    And I am on the admin portal sign in page
+    When I fill in the Username and Password fields with my authenticated credentials "hmcts.super.fact@gmail.com" "Pa55word11"
+    And click the Sign In button
+    Then I can view the courts or tribunals in a list format
+    And they are in alphabetical order
+    When I click edit next to a chosen court or tribunal
+    Then I am redirected to the Edit Court page for the chosen court
+    When I click the opening hours tab
+    Then I can view the existing opening hours
+
+  Scenario: Add new opening hours
+    When I enter new opening hours entry
+    And I click the Add button
+    And I click save
+    Then a green update message is displayed
```

Comment:
Is it possible to check the new entry which has been added? And to check  that the text has been displayed as expected? I would imagine the step would need to be able to see the new element...so it would have to reload the page or something...

We can chat about this one if you want

Reply:
Added a further step in updated code.